stdlib/LinearAlgebra/src/symmetriceigen.jl

```diff
@@ -195,3 +219,32 @@ function eigvals!(A::Hermitian{T,S}, B::Hermitian{T,S}; sortby::Union{Function,N
     return vals
 end
 eigvecs(A::HermOrSym) = eigvecs(eigen(A))
+
+# Note: No specilized LAPACK routines for Matrix+Symmetric and Symmetric+Matrix exist. Hence, the calls are forwarded to conventional Matrix eigvals!
+function eigvals(A::StridedMatrix{T}, B::Union{RealHermSymComplexHerm{T},Diagonal{T}}; sortby::Union{Function,Nothing}=nothing) where {T<:Number}
+    return eigvals(A, Matrix{T}(B); sortby)
```

Comment:
But these can call `eigvals!`, can't they? After we converted `B` to `Matrix`? The idea is that one calls `eigen` or `eigvals`, and they do all that is needed in one call to then do it in-place. Same a few lines below.

Reply:
If we call `eigvals!` or `eigen!` then we may need to make copies of (in this case) `A` or `B`. I am also equally fine for this solution, if you feel this would be better...

Reply:
@dkarrasch @jishnub Commit: https://github.com/aravindh-krishnamoorthy/julia/commit/d186845b1b429aa1bced005d39c105cf280dc410

Please feel free to let me know if you feel more changes are necessary. I will be happy to align the code to the general conventions of the Julia community.